policy/simplerecipe_test_proto.js

```diff
@@ -0,0 +1,118 @@
+/*
```

Comment:
Presuming this is the more concise version of the proto constructed in JSON beneath, but worth noting that in a comment, I wasn't sure at first, thought maybe it was some commented out other code reference or something.

Reply:
This is the IR output that Raksha consumes, this proto is the wire format that represents this IR. I mean, in theory we don't have to use protos and could just send text IRs to be parsed, but then it's more error prone and requires parsing.